_data/pascaladhikary/pascaladhikary.md

```diff
@@ -0,0 +1,7 @@
+---
+name: pascaladhikary
+institution: Illinois Mathematics & Science Academy
+profile_pic: mona.png
```

Comment:
The name here and the name of your image do not match.

Also, the image should be of 1:1 ratio and at least 544x544.

You have also not updated your checklist. To do so, simply insert "x" in between "[" and "]".

For example:
[x]

You should update these things in order to make it easier for moderators to review your PR and merge.